.github/workflows/build.yml

```diff
@@ -0,0 +1,30 @@
+name: build
+
+on:
+  push:
+    branches: main
+  pull_request:
+    branches: main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
We should update notation [build.yml](https://github.com/notaryproject/notation/blob/main/.github/workflows/build.yml#L11) to use the same.

Reply:
Can we also build/test on Windows and Mac?

Reply:
Make sense. But we may want to do this in a seperate PR, as currently we use make test to generate coverage report, which will not work on Windows.

Reply:
@gokarnm Updated notation workflow. See https://github.com/notaryproject/notation/pull/212/files